components/script/dom/bluetoothremotegattcharacteristic.rs

```diff
@@ -167,18 +187,32 @@ impl BluetoothRemoteGATTCharacteristicMethods for BluetoothRemoteGATTCharacteris
     fn ReadValue(&self) -> Rc<Promise> {
         let p = Promise::new(&self.global());
         let p_cx = p.global().get_cx();
+
+        // Step 1.
         if uuid_is_blocklisted(self.uuid.as_ref(), Blocklist::Reads) {
             p.reject_error(p_cx, Security);
             return p;
         }
+
+        // Step 2.
         if !self.Service().Device().Gatt().Connected() {
             p.reject_error(p_cx, Network);
             return p;
         }
+
+        // TODO: Step 3 - 4: Implement representedCharacteristic internal slot for BluetoothRemoteGATTCharacteristic.
+
+        // TODO: Step 5: Implement the `connection-checking-wrapper` algorithm for BluetoothRemoteGATTServer.
+
+        // Step 5.1.
+        // Note: May make this check in the lower level too.
```

Comment:
Because the `properties` link in [ReadValue](https://webbluetoothcg.github.io/web-bluetooth/#dom-bluetoothremotegattcharacteristic-readvalue) Step 5.1. points [here](https://webbluetoothcg.github.io/web-bluetooth/#characteristic-properties) not to the [BluetoothCharacteristicProperties](https://webbluetoothcg.github.io/web-bluetooth/#characteristicproperties)